pycalphad/model.py

```diff
@@ -111,7 +112,13 @@ def wrap_symbol(obj):
             else:
                 return Symbol(obj)
         if parameters is not None:
-            symbols.update([(wrap_symbol(s), val) for s, val in parameters.items()])
+            if isinstance(parameters, dict):
+                symbols.update([(wrap_symbol(s), val) for s, val in parameters.items()])
+            else:
+                # Lists of symbols that should remain symbolic
+                for s in parameters:
+                    symbols.pop(s)
```

Comment:
This should probably be `symbols.pop(wrap_symbol(s))` in case strings are passed.